### ISSUE TITLE
How to recover coins 

### ISSUE THREAD

**Issue by saawan77** [vergecurrency/verge#1037]
# Issue Template

## Context
Hello Guy's,
I posted this sometime ago but was not able to get it fixed due to some complications in my personal life. 
I want to thank the devs who commented on it before and i hope they will help me once more, very sorry to bother you guy's. This is the situation.
I cannot acces my xvg electrum wallet anymore.
The wallet is not synchronizing. I have the 2.4.1 xvg wallet on windows 10.
The only option i see on the Verge website is the Windows Tor QT Wallet.
Do i need to download the tor browser for that wallet ? 
Can someone please explain me in layman's terms how to get a new wallet running and recover my coins from the old wallet ?
I have my seed from the old electrum xvg wallet .

 

## Prerequisites

* [ ] Are you running the latest version?

## User Story:

* "I'm submitting a ... "
  - [ ] bug report
  - [ ] feature request
  - [ x] support request 
  - [ ] translation request 

## Steps to Reproduce (for bugs)

* Steps :
1.
2.
3.
4.

## Your Environment

* Verge Release:
* CPU:
* GPU:
* RAM:
* OS:
    - [ ] I'm using Mac
    - [ ] I'm using Linux
    - [ ] I'm using Mac

**Comment by jrsikors** [comment]
Do you only have the seed?  Or do you have private key(s) to the addresses as well?

If you only have seed, then you need to use Electrum.

Have you followed instructions here on setting up / running the Tor Electrum wallet?

https://verge.zendesk.com/hc/en-us/articles/360015942172--Tor-Electrum-XVG-Beginner-s-Guide

**Comment by jrsikors** [comment]
Couple other guides that might help as well:
https://verge.zendesk.com/hc/en-us/articles/360016162331--TOR-No-Connections-Not-Connected

https://verge.zendesk.com/hc/en-us/articles/360016162451--TOR-Seed-restore-not-working

**Comment by saawan77** [comment]
Is the tor electrum working ?

**Comment by saawan77** [comment]
I cannot see any tor electrum wallet on the  verge website. 
is it still supported ?

**Comment by jrsikors** [comment]
I just tried the Tor electrum wallet and it connected to the network just fine.  Install instructions and link are on that first link I sent.  Don't forget to install the tor browser and run it as well.

I would recommend sending your coins to one of the preferred wallets after you get it sorted it out.

**Comment by saawan77** [comment]
oke, i will try to do so. i allso got to export the private keys.
Do i have to run the tor browser and then unzip the tor exe file ?

**Comment by jrsikors** [comment]
do you have an iphone or droid?  on the iPhone there is an option to import ,, and your electrum seed usually works in there.

Step 1) Install Tor browser
Step 2)  Unzip Tor electrum
Step 3)  Run Tor Browser
Step 4)  Run Tor Electrum

Steps 1 and 2 don't matter which order necedssarily, but steps 3 and 4 definitely do.   Running tor browser gets you on the Tor network.  So you want to do that before you try to run the Tor electrum client.  Otherwise you wont be able to connect to any Tor servers if you aren't connected to Tor network.

**Comment by saawan77** [comment]
it appears that i allready had an electrum tor installed, probably when i was installing the normal electrum in the past . the password and seed are allso correct. tor browser is running and the tor wallet is also running . but because this is already an installed tor wallet i guess i can start it as a new wallet ?

**Comment by saawan77** [comment]
i didn't ever use the tor electrum wallet, there are no coins on it. do i have to delete it and install a new tor wallet ?

**Comment by jrsikors** [comment]
If all of your coins are on one address, and you now have the private key for that address, I would suggest just importing that one private key into one of the other wallets.  

I think Electrum Tor and Electrum Clearnet share different data directories.  So yeah if you started up E.Tor for the first time, it wouldn't recognize your wallet from E.Clearnet, and you'd have to make a new one and import seed or private keys.

What is your ultimate goal here though?  Are you just making sure you have access to them?  Or are you trying to get them to an exchange? 

I would still suggest getting importing your wallet address(es) with your coins into one of the more supported wallets.

**Comment by saawan77** [comment]
I am only trying to get access to the coins not to get them to an exchange. 
It would be better to get the coins to the Windows Tor Qt Wallet, but i dont know how to import the adresses from the old electrum wallet to the new Qt wallet. 
Can you guide me trough it ?

**Comment by saawan77** [comment]
By the way, i'm very happy that you are helping me here with patience  :)
I'm not very technical with computers

**Comment by marpme** [comment]
1. Open electrum
2. Open this special menu: ![image](https://user-images.githubusercontent.com/7430964/84765804-0b8bef80-afd0-11ea-807d-8745cd046e43.png)
3. Look for the address(es) that contain coins 
4. Copy the **private key** like shown:
![image](https://user-images.githubusercontent.com/7430964/84765880-2b231800-afd0-11ea-930b-f679ca4e779c.png)
5. Follow the [guide on importing them in the QT wallet](https://bitcoin.stackexchange.com/a/5944).
